src/mme-app/conf/log.json

```diff
@@ -0,0 +1,90 @@
+{
+    "EpcTools": {
```

Comment:
Need to discuss this.... getting in big dependency through logging framework + etc tools is worry to me. We already have basic logging framework, its good to use existing one.

Reply:
But, we are not using logging framework of epctool. This file comes with epctool module.